### ISSUE TITLE
Wish detail: Save images/videos

### ISSUE THREAD

**Issue by jrgoodpasture** [homedepot/infinite-wish-board#10]
Proposal to add Save images/videos to to the site. 

**Feature Name** 
Save images/videos

**Functionality**

**Notes**

**Comment by alex-hall** [comment]
@fzondlo , how did you solve the saving of large files in MongoDB?

**Comment by alex-hall** [comment]
@WRMilling , how should we store these files? Probably not in MongoDB.

**Comment by fzondlo** [comment]
Yeah, I would recommend AWS/GCP file storage and just storing the URL in the database. For the hackathon I'm not sure we ever got the images to upload into the database. Let me know if you guys would like any help on this or other features.

**Comment by aclaire-thd** [comment]
Q: should the prompt say just image? or "media" (to include video)?

Looks like the prompt to add media is on the wish detail summary

<img width="1540" alt="WishBoard-WishDetailSummary" src="https://user-images.githubusercontent.com/48930585/66015753-d2dbd280-e4a1-11e9-82df-f858d195aa0a.png">

But should there be a working link on the wish curation list?

<img width="1562" alt="WishBoard-WishList-AddImage" src="https://user-images.githubusercontent.com/48930585/66015739-c6f01080-e4a1-11e9-8f0c-54bb45e75002.png">
